frontend/src/App.js

```diff
@@ -53,6 +54,10 @@ function App() {
               path="/admin/category/:id/questions"
               element={<AdminQuestions />}
             />
+            <Route
+              path="/admin/category/:id/addWord"
+              element={<AdminAddWord />}
+            />
```

Comment:
```suggestion
            <Route
              path="/admin/category/:id/question/add"
              element={<AdminAddWord />}
            />
```
this is add question so url must be like this.
also dont combine the type and action

Best urls for frontend samples
```
/questions        //list
/question/id      //view 1 question
/question/add   //add a question
/question/id/edit //edit form of question
```